### ISSUE TITLE
GNome 46 theme not applied on toolbar 

### ISSUE THREAD

**Issue by radudiaconu0** [vinceliuice/Orchis-theme#425]
![image](https://github.com/vinceliuice/Orchis-theme/assets/52667211/5be4f86d-f964-4c1b-8099-8ca2d00b63e1)
![image](https://github.com/vinceliuice/Orchis-theme/assets/52667211/2770f5b1-3a43-4801-b0b6-2b21b4ea982c)

i followed the steps to install the theme and i fdont have the floting toolbar and the them eis not applied there at all. 
i am on fedora 40 gnome 46.1

`./install.sh -c dark -t default -l -s standard --tweaks black macos submenu`

**Comment by vinceliuice** [comment]
You need install `gnome-tweaks` and `user-themes` extension : https://extensions.gnome.org/extension/19/user-themes/

**Comment by vinceliuice** [comment]
Use gnome-tweaks to change the theme
![Screenshot from 2024-04-25 10-53-02](https://github.com/vinceliuice/Orchis-theme/assets/7604295/dd9afabc-bd71-4913-b730-8d61bd0884d9)

**Comment by radudiaconu0** [comment]
> Use gnome-tweaks to change the theme
> ![Screenshot from 2024-04-25 10-53-02](https://github.com/vinceliuice/Orchis-theme/assets/7604295/dd9afabc-bd71-4913-b730-8d61bd0884d9)
> 

Already did that.

**Comment by vinceliuice** [comment]
You changed the shell theme but nothing happen ?

**Comment by radudiaconu0** [comment]
yeah now it works. appprently icant change shell theme from gnome tweaks. i have to do it from user theemes extensions settings

**Comment by radudiaconu0** [comment]
i combine your theme with blur my shell
